### PR TITLE
watch example

### PR DESCRIPTION
**Pull request by wojons**
Readme example of how to watch in forego

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,6 +101,21 @@ if err := f.Remove(); err != nil {
 }
 ```
 
+### Watch
+
+```go
+notifications := make(chan firego.Event)
+if err := f.Watch(notifications); err != nil {
+	log.Fatel(err)
```
Comment (MrGossett) [inline README.md:109]: typo :smile:

should be `log.Fatal(err)`

File: README.md [hunk 2]
```diff
@@ -101,6 +101,21 @@ if err := f.Remove(); err != nil {
...
+}
+defer func() { f.StopWatching() }()
```
Comment (MrGossett) [inline README.md:111]: this should be `defer f.StopWatching()`

File: README.md [hunk 3]
```diff
@@ -101,6 +101,21 @@ if err := f.Remove(); err != nil {
...
+for {
+	event, ok := <-notifications
+	fmt.Println(event)
+	fmt.Println(ok)
```
Comment (MrGossett) [inline README.md:115]: this loop would continue to spin forever, even if `notifications` is closed. let's add this at the end:

``` go
if !ok {
    break
}
```

**Comment by zabawaba99** [comment]
There is an example of `Watch` and `StopWatching` in the [GoDocs](http://godoc.org/github.com/CloudCom/firego) as well. It would probably be a good idea to make sure they are in sync.

**Comment by wojons** [comment]
I am fine with all the changes you gave let me check whats in the godoc i was able to figure out how this worked using your unit test.

**Pull request closed without merging**